py/jupyterlite/src/jupyterlite/addons/base.py

```diff
@@ -33,6 +33,11 @@ class BaseAddon(LoggingConfigurable):
 
     manager: LiteManager = Instance(LiteManager)
 
+    def __init__(self, manager, *args, **kwargs):
+        kwargs["parent"] = manager
+        kwargs["manager"] = manager
+        super().__init__(*args, **kwargs)
+
```

Comment:
@datakurre in the near term you can  just apply this to your addon, if you need to...